src/styles/main.css

```diff
@@ -0,0 +1,233 @@
+  content: "4";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(5)::before {
+  content: "5";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(6)::before {
+  content: "6";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(7)::before {
+  content: "7";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(8)::before {
+  content: "8";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(9)::before {
+  content: "9";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(10)::before {
+  content: "10";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(11)::before {
+  content: "11";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(12)::before {
+  content: "12";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(13)::before {
+  content: "13";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(14)::before {
+  content: "14";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(15)::before {
+  content: "15";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(16)::before {
+  content: "16";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(17)::before {
+  content: "17";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(18)::before {
+  content: "18";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(19)::before {
+  content: "19";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(20)::before {
+  content: "20";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(21)::before {
+  content: "21";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(22)::before {
+  content: "22";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(23)::before {
+  content: "23";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(24)::before {
+  content: "24";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(25)::before {
+  content: "25";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(26)::before {
+  content: "26";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(27)::before {
+  content: "27";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(28)::before {
+  content: "28";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(29)::before {
+  content: "29";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(30)::before {
+  content: "30";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(31)::before {
+  content: "31";
+  font-size: 30px;
+}
+
+.calendar--start-mon .calendar__day:first-child {
+  margin-left: 102px;
+}
+
+.calendar--start-tue .calendar__day:first-child {
+  margin-left: 205px;
+}
+
+.calendar--start-wed .calendar__day:first-child {
+  margin-left: 308px;
+}
+
+.calendar--start-thu .calendar__day:first-child {
+  margin-left: 411px;
+}
+
+.calendar--start-fri .calendar__day:first-child {
+  margin-left: 514px;
+}
+
+.calendar--start-sat .calendar__day:first-child {
+  margin-left: 617px;
+}
+
+.calendar--start-sun .calendar__day:first-child {
+  margin-left: 720px;
+}
+
+.calendar--month-length-28 .calendar__day:nth-child(n+29) {
+  display: none;
+}
+
+.calendar--month-length-29 .calendar__day:nth-child(n+30) {
+  display: none;
```

Comment:
Same with modifiers. Make use of SASS features
https://sass-lang.com/documentation/at-rules/control/each